sale_automatic_workflow/tests/test_flow.py

```diff
@@ -71,7 +54,7 @@ def progress(self):
     def test_full_automatic(self):
         workflow = self._create_full_automatic()
         sale = self._create_sale_order(workflow)
-        sale.onchange_workflow_process_id()
```

Comment:
This lines are failing on tests, function is "onchange_workflow_process_id" not "_onchange_workflow_process_id"